cmd/nginx-ingress/main.go

```diff
@@ -1197,6 +1197,36 @@ func initLogger(logFormat string, level slog.Level, out io.Writer) context.Conte
 					a.Value = slog.AnyValue(src)
 				}
 			}
+			// Handle custom timestamp formatting
+			if a.Key == slog.TimeKey {
+				if t, ok := a.Value.Any().(time.Time); ok {
+					switch timeFormat {
+					case "unix":
+						// Unix timestamp in seconds
+						return slog.Attr{
+							Key:   slog.TimeKey,
+							Value: slog.Int64Value(t.Unix()),
+						}
+					case "unix-ms":
+						// Unix timestamp with milliseconds
+						return slog.Attr{
+							Key:   slog.TimeKey,
+							Value: slog.Int64Value(t.UnixMilli()),
+						}
+					case "unix-ns":
+						// Unix timestamp with nanoseconds
+						return slog.Attr{
+							Key:   slog.TimeKey,
+							Value: slog.Int64Value(t.UnixNano()),
+						}
+					case "default":
```

Comment:
this isn't needed here, because

1. either we've already made sure that the time log format is one of the four allowed values, in which case if it's not the other three, then the switch's default arm is enough, or
2. we haven't, and if we pass in some weird value like `iso8601`, which should not be allowed, that will also be ignored and `rfc3339` will be printed, as per the original value coming from slog